### ISSUE TITLE
Investigate if `offset.AdjustForWhiteSpace(in text);` would be efficient in places where we currently call `continue`

### ISSUE THREAD

**Issue by Marvin-Brouwer** [Marvin-Brouwer/FluentSerializer#81]
It is arguably a more readable approach if you compare this:

```csharp
while (!tagFinished && text.WithinCapacity(in offset))
{
	if (text.HasCharactersAtOffset(in offset,
		XmlCharacterConstants.TagTerminationCharacter,
		XmlCharacterConstants.TagEndCharacter))
	{
		offset.AdjustForToken(XmlCharacterConstants.TagTerminationCharacter);
		elementClosed = true;
		offset.AdjustForToken(XmlCharacterConstants.TagEndCharacter);
		break;
	}
	if (text.HasCharacterAtOffset(in offset, XmlCharacterConstants.TagEndCharacter))
	{
		offset.AdjustForToken(XmlCharacterConstants.TagEndCharacter);
		break;
	}

	if (text.HasWhitespaceAtOffset(in offset))
	{
		offset++;
		continue;
	}

	_attributes.Add(new XmlAttribute(in text, ref offset));
}
```

With this:

```csharp
while (!tagFinished && text.WithinCapacity(in offset))
{
	if (text.HasCharactersAtOffset(in offset,
		XmlCharacterConstants.TagTerminationCharacter,
		XmlCharacterConstants.TagEndCharacter))
	{
		offset.AdjustForToken(XmlCharacterConstants.TagTerminationCharacter);
		elementClosed = true;
		offset.AdjustForToken(XmlCharacterConstants.TagEndCharacter);
		break;
	}
	if (text.HasCharacterAtOffset(in offset, XmlCharacterConstants.TagEndCharacter))
	{
		offset.AdjustForToken(XmlCharacterConstants.TagEndCharacter);
		break;
	}

	offset.AdjustForWhiteSpace(in text);

	_attributes.Add(new XmlAttribute(in text, ref offset));
}
```

However the question is if having a loop inside of a loop impacts performance.

**Comment by Marvin-Brouwer** [comment]
For the results see: #133

**Comment by Marvin-Brouwer** [comment]
Closed by #133 
Ended up only being used once though.
